app/assets/javascripts/plannings.js

```diff
@@ -157,29 +156,29 @@ const spreadsheetModalExport = function(columns, planningId) {
   }).mouseleave(function(evt) {
     $('a.remove', evt.currentTarget).hide();
   });
-  if (localStorage.spreadsheetFormat)
-    $('[name=spreadsheet-format][value=' + localStorage.spreadsheetFormat + ']').prop('checked', true);
-
+  if (export_settings && export_settings['format']) {
+    $('[name=spreadsheet-format][value=' + export_settings['format'] + ']').prop('checked', true);
+  }
   $('#btn-spreadsheet').click(function() {
     var planningsId = getPlanningsId();
     if (!planningId && planningsId.length == 0) {
       warning(I18n.t('plannings.index.export.none_planning'));
       return;
     }
 
-    var spreadsheetStops = localStorage.spreadsheetStops = $('.spreadsheet-stops:checked').map(function(i, e) {
+    var spreadsheetStops = $('.spreadsheet-stops:checked').map(function(i, e) {
       return $(e).val();
     }).get().join('|');
-    var spreadsheetColumnsExport = localStorage.spreadsheetColumnsExport = $('#columns-export').find('li').map(function(i, e) {
+    var spreadsheetColumnsExport = $('#columns-export').find('li').map(function(i, e) {
       return $(e).attr('data-value');
     }).get().join('|');
-    var spreadsheetColumnsSkip = localStorage.spreadsheetColumnsSkip = $('#columns-skip').find('li').map(function(i, e) {
+    var spreadsheetColumnsSkip = $('#columns-skip').find('li').map(function(i, e) {
       return $(e).attr('data-value');
     }).get().join('|');
-    var spreadsheetFormat = localStorage.spreadsheetFormat = $('[name=spreadsheet-format]:checked').val();
+    var spreadsheetFormat = $('[name=spreadsheet-format]:checked').val();
     var basePath = $('[name=spreadsheet-route]').val() ? ('/routes/' + $('[name=spreadsheet-route]').val()) : (planningId) ? '/plannings/' + planningId : '/plannings';
 
-    window.location.href = basePath + '.' + spreadsheetFormat + '?stops=' + spreadsheetStops + '&columns=' + spreadsheetColumnsExport + "&ids=" + planningsId;
+    window.location.href = basePath + '.' + spreadsheetFormat + '?stops=' + spreadsheetStops + '&columns=' + spreadsheetColumnsExport + "&ids=" + planningsId + '&columns_skip=' + spreadsheetColumnsSkip + '&additional_stops=' + spreadsheetStops;
```

Comment:
```suggestion
    window.location.href = basePath + '.' + spreadsheetFormat + '?stops=' + spreadsheetStops + '&columns=' + spreadsheetColumnsExport + "&ids=" + planningsId + '&skips=' + spreadsheetColumnsSkip
```

Reply:
`additional_stops` are already provided through `stops`
`columns_skip` could be reduced to `skips`